lib/maths/common/unittest/CNaiveBayesTest.cc

```diff
@@ -270,6 +281,40 @@ BOOST_AUTO_TEST_CASE(testPropagationByTime) {
     }
 }
 
+BOOST_AUTO_TEST_CASE(testExtrapolation) {
+    // Test that we detect when we're extrapolating the class conditional
+    // distributions to predict a class labels.
```

Comment:
When reading through the test, it's difficult to see where exactly the extrapolation detection is asserted. Can you maybe make it more clear?

Reply:
Oh actually this is a mistake. I was testing this in a different way in an earlier incarnation and this comment got left behind. I fixed.